File: typescript/src/Submarine.ts
import {SubmarineInterface} from "@submarine-domain/SubmarineInterface";

export class Submarine implements SubmarineInterface
{
    public executeCommand = (command: string) => {
        throw new Error('Not implemented')
    };
    public aim = (): number => {
        throw new Error('Not implemented')
    };
    public position = (): number => {
        throw new Error('Not implemented')
    };
    public depth = (): number => {
        throw new Error('Not implemented')
    };
}

File: python/src/submarine.py
from src.submarine_interface import SubmarineInterface


class Submarine(SubmarineInterface):
    def execute_command(self, command: str) -> None:
        raise NotImplementedError("Not Implemented")

    def aim(self) -> int:
        raise NotImplementedError("Not Implemented")

    def position(self) -> int:
        raise NotImplementedError("Not Implemented")

    def depth(self) -> int:
        raise NotImplementedError("Not Implemented")
